--- orig-sources/com/terraformersmc/modmenu/gui/widget/LegacyTexturedButtonWidget.java
+++ sources/com/terraformersmc/modmenu/gui/widget/LegacyTexturedButtonWidget.java
@@ -1,5 +1,6 @@
 package com.terraformersmc.modmenu.gui.widget;
 
+import net.minecraft.client.gl.RenderPipelines;
 import net.minecraft.client.gui.DrawContext;
 import net.minecraft.client.gui.widget.ButtonWidget;
 import net.minecraft.client.gui.widget.TexturedButtonWidget;
@@ -53,7 +54,7 @@
 		}
 
 		context.drawTexture(
-			RenderLayer::getGuiTextured,
+			RenderPipelines.GUI_TEXTURED,
 			this.texture,
 			this.getX(),
 			this.getY(),
