--- orig-sources/com/terraformersmc/modmenu/gui/widget/entries/ModListEntry.java
+++ sources/com/terraformersmc/modmenu/gui/widget/entries/ModListEntry.java
@@ -12,6 +12,7 @@
 import com.terraformersmc.modmenu.util.mod.ModBadgeRenderer;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gl.RenderPipelines;
 import net.minecraft.client.gui.DrawContext;
 import net.minecraft.client.gui.widget.AlwaysSelectedEntryListWidget;
 import net.minecraft.client.render.RenderLayer;
@@ -68,9 +69,8 @@
 			DrawingUtil.drawRandomVersionBackground(mod, drawContext, x, y, iconSize, iconSize);
 		}
 
-		RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F);
 		GlStateManager._enableBlend();
-		drawContext.drawTexture(RenderLayer::getGuiTextured, this.getIconTexture(), x, y, 0.0F, 0.0F, iconSize, iconSize, iconSize, iconSize);
+		drawContext.drawTexture(RenderPipelines.GUI_TEXTURED, this.getIconTexture(), x, y, 0.0F, 0.0F, iconSize, iconSize, iconSize, iconSize);
 		GlStateManager._disableBlend();
 
 		Text name = Text.literal(mod.getTranslatedName());
@@ -137,30 +137,34 @@
 				boolean hoveringIcon = mouseX - x < iconSize;
 				if (this.list.getParent().modScreenErrors.containsKey(modId)) {
 					drawContext.drawGuiTexture(
-						RenderLayer::getGuiTextured,
-						hoveringIcon ? ERROR_HIGHLIGHTED_ICON : ERROR_ICON,
-						x,
-						y,
-						iconSize,
-						iconSize
+							RenderPipelines.GUI_TEXTURED,
+							hoveringIcon ? ERROR_HIGHLIGHTED_ICON : ERROR_ICON,
+							x,
+							y,
+							iconSize,
+							iconSize
 					);
 					if (hoveringIcon) {
 						Throwable e = this.list.getParent().modScreenErrors.get(modId);
-						this.list.getParent().setTooltip(this.client.textRenderer.wrapLines(ModMenuScreenTexts.configureError(modId, e), 175));
+						drawContext.drawTooltip(
+								this.client.textRenderer.wrapLines(ModMenuScreenTexts.configureError(modId, e), 175),
+								mouseX,
+								mouseY
+						);
 					}
 				} else {
 					int v = hoveringIcon ? iconSize : 0;
 					drawContext.drawTexture(
-						RenderLayer::getGuiTextured,
-						MOD_CONFIGURATION_ICON,
-						x,
-						y,
-						0.0F,
-						(float) v,
-						iconSize,
-						iconSize,
-						textureSize,
-						textureSize
+							RenderPipelines.GUI_TEXTURED,
+							MOD_CONFIGURATION_ICON,
+							x,
+							y,
+							0.0F,
+							(float) v,
+							iconSize,
+							iconSize,
+							textureSize,
+							textureSize
 					);
 				}
 			}
